app/models/api/task.rb

```diff
@@ -20,5 +20,11 @@ def late?
     def reporting_period
       [Date::MONTHNAMES[period_month], period_year].join(' ')
     end
+
+    def completed_at
```

Comment:
We're using different words to describe the same thing: "submitted_at" in the API but "completed_at" in the frontend application, which is making me feel uneasy. Is there a good reason why we're not calling it the same thing in both places to avoid confusion and misunderstandings between supplier users and CCS admin users?

Reply:
Oh, sorry, so this is the "task" that's "completed", whilst it was a submission that was "submitted"...